electron-app/magnifier/magnifier-main-rust.ts

```
@@ -0,0 +1,316 @@
+            `[Magnifier] Grid size change: ${oldGridSize} → ${this.gridSize}`
+          );
+
+          // Update grid size in Rust sampler
+          this.samplerManager.updateGridSize(this.gridSize);
+        }
+      });
+
+      globalShortcut.register('Escape', () => resolveOnce(null));
+
+      // Set up data callback for the sampler
+      const dataCallback = (pixelData: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)